### PR TITLE
null: DevWs

### PR DESCRIPTION
**Pull request by Duncankn**
Submitting 'DevWs' for publication

### CODE REVIEW

**Comment by shrakuma** [comment]
@bflorian please open this PR as this is not a Test PR

**Comment by shrakuma** [comment]
Hi @Duncankn
Thank you for working on integrating with our platform. We are reviewing the code and we shall keep you posted here and in the other PRs that you have submitted.

**Comment by MAblewicz** [comment]
Reopenedy by Shravan's  request

**Comment by Duncankn** [comment]
@MAblewiczS I put all devices into one publish request. Is that what you want? Thanks!

File: devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy [hunk 1]
```diff
@@ -48,6 +48,7 @@ metadata {
 		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "TRADFRI bulb E14 W op/ch 400lm", deviceJoinName: "IKEA TRADFRI LED Bulb"
 		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "TRADFRI transformer 10W", deviceJoinName: "TRÅDFRI Driver for wireless control 10W"
 		fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000", outClusters: "0005, 0019, 0020, 1000", manufacturer: "IKEA of Sweden", model: "TRADFRI transformer 30W", deviceJoinName: "TRADFRI Driver for wireless control 30W"
+        fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, FFFF", outClusters: "0019",manufacturer: "Megaman", model: "ZLL-DimmableLight", deviceJoinName: "INGENIUM ZB Dimmable Light"
```
Comment (shrakuma) [inline devicetypes/smartthings/zll-dimmer-bulb.src/zll-dimmer-bulb.groovy:51]: @Duncankn the indentation is not correct here. please fix the formatting.

**Comment by Duncankn** [comment]
This PR is redundance now, it can be closed, thanks!

**Pull request closed without merging**